exp/test_cases/isca_job.sh

```diff
@@ -1,17 +1,20 @@
-#!/bin/sh
-#PBS -V # export all environment variables to the batch job.
-#PBS -d . # set working directory to .
-#PBS -q ptq # submit to the parallel test queue
-#PBS -l nodes=1:ppn=8 # nodes=number of nodes required. ppn=number of processors per node
-#PBS -l walltime=0:30:00 # Maximum wall time for the job.
-#PBS -A Research_Project-PROJECTNUMBER # research project to submit under.
-#PBS -m e -M USER@exeter.ac.uk # email me at job completion
-
 # This is an example of how to submit an experiment to the queue on the Isca supercomputer
 # in Exeter.  USER and PROJECTNUMBER in the metadata above need to be
 # updated to appropriate values for your experiment.
 # For more information about running on the Exeter cluster, see the wiki
-# here: https://wiki.exeter.ac.uk/display/ISCA/Isca+Wiki
+# here: http://login.isca.ex.ac.uk:8080/wiki/Isca+Guide.pdf
```

Comment:
This link does not work for me. :/ But the old one does!

Reply:
Hmm, I can't get either of the links to work. Safari and chrome block the old one because there is no security certificate? If you got the old one to work shall we revert to that one? I checked the guide they emailed around September and that defo directs you to the new link.

Reply:
I did the sensible thing and emailed the isca support team!

Reply:
Try this one: https://universityofexeteruk.sharepoint.com/sites/ExeterARC
It's a series of word documents though.... lol